### ISSUE TITLE
this is undefined in componentDidUpdate

### ISSUE THREAD

**Issue by erin-doyle** [romeovs/react-a11y#16]
I've found that in a 3rd party component I'm using, in an application where I'm also using react-a11y-alt, `this` is coming up as undefined in the componentDidUpdate() function.  Using the component in other places does not have this issue.  

It looks as if the react-a11y-alt after.js render() function patches componentDidUpdate as well as componentDidMount and when it does so it replaces the context with undefined:

```javascript
original.apply(undefined, args);
```

It looks as if the component's context is available there and could be passed in as the `thisArg` instead of undefined, so I'm not sure why that isn't being done.

**Comment by erin-doyle** [comment]
Correction.  The actual source code is:

```javascript
original.apply(this, args);
```

It was only being transpiled from `this` to `undefined`.  This looks to be a babel issue.
